README.md

```diff
@@ -12,123 +12,62 @@ This plugin provides an authentication with [passportjs strategies](http://passp
 
 # Configuration
 
-You can override the provided default configuration using [Kuzzle command-line interface](http://kuzzle.io/guide/#managing-plugins-using-the-cli)
+To edit the configuration of a plugin see [custom plugin configuration](http://docs.kuzzle.io/plugin-reference/#custom-plugin-configuration).
 
 List of available configurations:
 
 | Name | Default value | Type | Description                 |
 |------|---------------|-----------|-----------------------------|
 | ``strategies`` | ``[]`` | Array | List of the providers you want to use with passport |
+| ``credentials`` | ``{}`` | Object | Credentials provided by the provider |
 | ``persist`` | ``{}`` | Object | Attributes you want to persist if the user doesn't exist |
 | ``scope`` | ``[]`` | Array | List of fields in the OAUTH 2.0 scope of access |
-| ``profile`` | ``"default"`` | Array | Profile of the new persisted user |
+| ``useAdId`` | | String | Attribute from the profile of the provider to use as Id if you want to persist the user in Kuzzle |
+| ``defaultProfile`` | ``"default"`` | Array | Profiles of the new persisted user |
+| ``mapToKuzzle`` | ```` | Object | Mapping of attributes to persist in the user persisted in Kuzzle |
```

Comment:
Nitpicking: `attributesMapping`or `kuzzleAttributesMapping` would be more explicit